src/interfaces/permissions-request.ts

```diff
@@ -3,18 +3,18 @@ import type { PermissionConditions, PermissionScope } from '../types/permissions
 import type { PermissionsRequestDescriptor, PermissionsRequestMessage } from '../types/permissions-types.js';
 
 import { getCurrentTimeInHighPrecision } from '../utils/time.js';
-import { v4 as uuidv4 } from 'uuid';
+import { removeUndefinedProperties } from '../utils/object.js';
 import { validateAuthorizationIntegrity } from '../core/auth.js';
 import { DwnInterfaceName, DwnMethodName, Message } from '../core/message.js';
 
 type PermissionsRequestOptions = {
   dateCreated?: string;
-  conditions?: PermissionConditions;
   description: string;
```

Comment:
Sanity: a bit odd that `description` in `descriptor` is optional but we require it in the SDK. Should we make it optional for consistency?

Reply:
Ope, good catch. Updated to make it optional everywhere.